tests/unittest_inference.py

```diff
@@ -1117,6 +1117,12 @@ def __radd__(self, other): return NotImplemented
         first = next(ast_node.infer())
         self.assertEqual(first, util.Uninferable)
 
+    @pytest.mark.filterwarnings("error::DeprecationWarning")
```

Comment:
Do we need to filter that ? I thought the fix was precisely to remove the deprecation warning ?

Reply:
Good Q, "filter" is a little misleading here. It's taking DeprecationWarning and raising it to `error`. Revert the fix and this test will fail.

Reply:
Otherwise the test would pass anyway, since `pytest` doesn't fail tests when there are deprecationwarnings.

Reply:
We might as well change the global configuration, we often have to so "fix deprecation warning" merge request both for astroid and pylint.